.pre-commit-config.yaml

```diff
@@ -29,3 +29,11 @@ repos:
     rev: v0.9.1
     hooks:
       - id: sphinx-lint
+
+# Configuration for codespell is in pyproject.toml
+- repo: https://github.com/codespell-project/codespell
+  rev: v2.2.6
+  hooks:
+  - id: codespell
+    additional_dependencies:
+    - tomli
```

Comment:
Please check your indentation. This stanza appears to be outdented by 2 spaces, resulting in pre-commit throwing an error.

Reply:
sorry about that... I added a check on formatting only later to my script.  I have fixed up that commit, rebased (thus merge of mater would no longer be needed), redone fixing of typos, force pushing now.

Reply:
FWIW, verified now that pre-commit works and detects a typo locally

```shell
❯ echo enchancement >> README.* && git commit -m 'TEMP: added a typo to check if caught' -a
[INFO] Initializing environment for https://github.com/psf/black.
[INFO] Initializing environment for https://github.com/sphinx-contrib/sphinx-lint.
[INFO] Installing environment for https://github.com/psf/black.
[INFO] Once installed this environment will be reused.
[INFO] This may take a few minutes...
[INFO] Installing environment for https://github.com/PyCQA/isort.
[INFO] Once installed this environment will be reused.
[INFO] This may take a few minutes...
[INFO] Installing environment for https://github.com/PyCQA/flake8.
[INFO] Once installed this environment will be reused.
[INFO] This may take a few minutes...
[INFO] Installing environment for https://github.com/sphinx-contrib/sphinx-lint.
[INFO] Once installed this environment will be reused.
[INFO] This may take a few minutes...
black................................................(no files to check)Skipped
check python ast.....................................(no files to check)Skipped
trim trailing whitespace.................................................Passed
check for merge conflicts................................................Passed
check json...........................................(no files to check)Skipped
check xml............................................(no files to check)Skipped
check yaml...........................................(no files to check)Skipped
mixed line ending........................................................Passed
isort................................................(no files to check)Skipped
flake8...............................................(no files to check)Skipped
Sphinx Lint..............................................................Passed
codespell................................................................Failed
- hook id: codespell
- exit code: 65

README.rst:139: enchancement ==> enhancement, enchantment

git commit -m 'TEMP: added a typo to check if caught' -a  7.97s user 1.21s system 74% cpu 12.369 total
```